src/coreclr/jit/unwind.cpp

```diff
@@ -203,19 +197,20 @@ void Compiler::unwindBegPrologCFI()
 
 void Compiler::unwindPushPopMaskCFI(regMaskTP regMask, bool isFloat)
```

Comment:
Review note: This method is only ever called for `TARGET_ARM`. The order of registers was opposite to the one expected in DWARF. Along with the change in `unwindPushPopCFI` it now fixes the ARM target to generate correct DWARF descriptors.

Both of these issues were previously masked by the counter part code in [ObjWriter](https://github.com/dotnet/llvm-project/blob/d8bacb4031b1d1e290ab2792e8378560419ee0de/llvm/tools/objwriter/objwriter.cpp#L1211-L1265) that converted DWARF codes to EHABI. Notably it reversed the register order at each code offset, and the `.regsave` opcode in the EHABI assembly automatically moved the call frame pointer by the register size.